lib/deliver/loader.rb

```diff
@@ -0,0 +1,13 @@
+require 'fastlane_core/languages'
+
+module Deliver
+  module Loader
```

Comment:
Maybe you could add a one line comment to the module `Loader` of what it is for :+1: